src/Command/PolicyCommand.php

```diff
@@ -76,7 +76,7 @@ public function templateData(Arguments $arguments): array
         $data = parent::templateData($arguments);
 
         $name = $arguments->getArgument('name');
-        if (empty($name)) {
+        if ($name === null || $name === '') {
```

Comment:
I don't think this is better but psalm thinks it is 🤷🏻

Reply:
`if (!$name)` was what I had in mind :)

Reply:
but psalm is not happy about that one as I described in core chat.

Because then `null` and `''` would be handled in the same if which "could lead to unexpected behavior" according to psalm.

Reply:
Thus the silencing on top of it :)

Reply:
Otherwise you would have to use 
```php
/** @var string|null $name */
$name = ...
if ($name === null) 
```
if you wanted to make this clean

Reply:
Well If I just look at https://www.php.net/manual/en/types.comparisons.php then we didn't really do anything here other than just suppress the warning...